bugbash.md

```diff
@@ -0,0 +1,2 @@
+# Bug Bash File For Activity
+who goes first?
```

Comment:
what do you think about making it `Who` instead of `who`

Reply:
good catch